### ISSUE TITLE
Устарел пейлоуд в сущности Future

### ISSUE THREAD

**Issue by toastyboost** [vitalets/tinkoff-invest-api#21]
Например, новый пейлоуд содержит ключ `initial_margin_on_buy`
 <img width="570" alt="Снимок экрана 2024-07-05 в 23 24 18" src="https://github.com/vitalets/tinkoff-invest-api/assets/51236614/5d93afff-08c1-4694-b74c-7e3b2d8e9568">

**Comment by vitalets** [comment]
Да, сейчас SDK отстает от текущих proto-файлов. По-хорошему бы обновиться на свежую версию, но там много breaking changes и это нужно прям сесть и заняться. Может у вас есть инфа по планам тинькоф насчет официального js sdk? 
Вот тут еще спрашивал https://github.com/vitalets/tinkoff-invest-api/issues/19#issuecomment-2044593704

**Comment by vitalets** [comment]
Завел ребятам тикет https://github.com/RussianInvestments/investAPI/issues/149

**Comment by vitalets** [comment]
Исправлено в версии 7.1.0 — поле `initialMarginOnBuy` теперь присутствует в сущности `Future` после обновления proto-контрактов.
